src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,240 @@
+and get advice on how to update them.
+[Best practices for dependency management][best practices]
+include using the most recent stable package versions,
+so you can get the latest bug fixes and improvements.
+
+{{site.alert.version-note}}
+  The `pub outdated` command was introduced in Dart 2.8.
+{{site.alert.end}}
+
+## Overview
+
+Here's how you can use `pub outdated` to help you
+update the dependencies of a package that you own
+(whether it's an app or library package):
+
+1. If your package doesn't have a `pubspec.lock` file
+   checked into source control,
+   **run `pub get`** in the top directory of the package —
+   the directory that contains your package's
+  [`pubspec.yaml`](/tools/pub/pubspec) file.
+1. **Run `pub outdated`**
+   to identify which package dependencies are out-of-date.
+   Note the affected packages,
+   so that later you can test the behavior of code that uses them.
+1. Follow the recommendations of `pub outdated` for updating the packages.
+   Some updates might require only running `pub upgrade`.
+   Others might require updating `pubspec.yaml` before running `pub upgrade`.
+1. **Run `pub outdated`** to confirm that you've updated all the dependencies.
+1. **Test** your package to confirm that it still works as expected.
+
+You might still have [transitive dependencies][] that are out-of-date.
+If you want to determine the cause,
+try running [`pub deps`][] and searching the output for
+the name of the out-of-date package.
+
+
+## Example
+
+Here's an example of running `pub outdated` on
+an example that depends on old versions of two packages:
+`args` and `pedantic`.
+As the following screenshot shows,
+`pub outdated` colorizes the output by default
+when you run it on the command line.
+
+![screenshot of pub outdated output; visible as text later in "Output columns" section](images/pub-outdated.png)
+
+The preceding output shows that one regular dependency (`args`)
+and one [dev dependency][] (`pedantic`) are out-of-date.
+For details on what each column shows, see the
+[output columns](#output-columns) section of this page.
+
+As a rule, you update the **dependencies** and **dev_dependencies** sections
+of your `pubspec.yaml` file
+so that each package uses the versions in the **Resolvable** column.
+In this example, the resolvable version of `args` is **1.6.0**,
+and the resolvable version of `pedantic` is **1.9.0**.
+
+{{site.alert.tip}}
+  To see what changed in the new version of a package
+  that's published on [pub.dev,]({{site.pub}})
+  look at the changelog in the package page.
+  For example, you can look at the **Changelog** tabs on the pages for the
+  [`args`][] and [`pedantic`][] packages.
+{{site.alert.end}}
+
+To fix the first dependency (`args`),
+all that's needed is to run `pub upgrade`:
+
+```terminal
+$ pub upgrade
+Resolving dependencies... 
+> args 1.6.0 (was 1.5.0)
+  pedantic 1.8.0 (1.9.0 available)
+Changed 1 dependency!
+```
+
+To fix the second dependency (`pedantic`),
```

Comment:
Maybe expand to "To fix the second dependency (`pedantic`), which is listed as resolveable," ?